Samples/Samples.Android/MainActivity.cs

```diff
@@ -25,6 +26,13 @@ protected override void OnCreate(Bundle bundle)
             LoadApplication(new App());
         }
 
+        protected override void OnResume()
+        {
+            base.OnResume();
+
+            Xamarin.Essentials.Platform.OnResume();
```

Comment:
We talked about this and it might be weird given we're only caring about the callback to the sort of 'main' activity of the app (or previous activity, the one which initiated the web request).  The lifecycle watcher keeps changing the current, so when the callback does happen, it would be the wrong activity.  This is really just meant to catch the cases where the user cancels out of the flow and we would otherwise be indefinitely waiting.  I think this is ok as is.